pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/ecdhes_key_template.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/aead"
+	commonpb "github.com/google/tink/proto/common_go_proto"
+	tinkpb "github.com/google/tink/proto/tink_go_proto"
+
+	ecdhespb "github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/proto/ecdhes_aead_go_proto"
+)
+
+// AES256GCMECDHESKeyTemplate is a KeyTemplate that generates an ECDH-ES P-256 key wrapping and AES256-GCM CEK with
```

Comment:
we will need a todo for chacha key wrapping template.

Reply:
TODO added... chacha key wrapping is tbd

Reply:
issue number?

Reply:
created #1637 and added it as a comment in the key templates 

We will need to discuss how chacha key wrapping will be done if it must be supported. But for now we can add Chacha content encryption with ECDE-ES key wrapping using a key with P-256 curve.

Reply:
created #1637 and added it as a comment in the key templates 

we will need to discuss how chacha key wrapping will be done if it must be supported. But for now we can add Chacha content encryption with ECDE-ES key wrapping using a key with P-256 curve